requirements/dev.txt

```diff
@@ -1,5 +1,7 @@
 -r default.txt
 
+appnope==0.1.0 \
+   --hash sha256:5b26757dc6f79a3b7dc9fab95359328d5747fcb2409d331ea66d0272b90ab2a0 
```

Comment:
Wasn't sure if you actually wanted to add this to the dependencies... this is what I had to add manually to get the dependencies installed on my machine..

Reply:
This makes sense. Our CI is all Linux, and I think everyone working on Normandy so far is using Linux either directly or via Docker. This looks like an OSX specific thing.

That being said, I *suspect* this line should probably be in `constraints.txt` instead of `dev.txt`. I don't think we actually want to ask pip to install it (for example, on Linux or CI systems), but if it does get installed we want to pin the version and hash.

That of course depends on the error you're getting. If moving it to the constraints file breaks things, lets keep it as is.

Reply:
Yes I assumed this was an OSX thing from #2 in the troubleshooting tips in the docs that suggested the platform difference could be the cause! Yay docs. It looks like I can move this to `constraints.txt` just fine.